src/stylesheets/components/_search.scss

```diff
@@ -25,12 +34,27 @@
       @include u-padding-x(2);
       width: auto;
     }
+
+    @media (forced-colors: active) {
+      border: 2px solid transparent;
+      @include place-icon($search-icon, "before");
+
+      &::before {
+        @include at-media("mobile-lg") {
+          content: none;
```

Comment:
# Search

The search component was missing the button outline as well as the magnifying glass icon in high contrast modes.

_Current High Contrast (left) / High Contrast Fix (right)_
![image](https://user-images.githubusercontent.com/25211650/160000454-c1d58084-a9e9-4eb8-953e-5e5903942d03.png)